uptane-standard.md

```diff
@@ -264,20 +275,20 @@ There are a number of factors that could impede the completion of the above scen
 The following topics will not be addressed in this document, as they represent threats outside the scope of Uptane:
 
 * Physical attacks, such as manual tampering with ECUs outside the vehicle.
-* Compromise of the supply chain (e.g., build system, version control system, packaging process). A number of strategies already (e.g., git signing, TPMs, in-toto {{IN-TOTO}}) exist to address this problem. Therefore, there is no need duplicate those techniques here.
+* Compromise of the supply chain (e.g., build system, version control system, packaging process). A number of strategies (e.g., git signing, TPMs, in-toto {{IN-TOTO}})  already exist to address this problem. Therefore, there is no need to duplicate those techniques here.
 * Problems associated with OBD or UDS programming of ECUs, such as authentication of communications between ECUs.
 
 ## Design Requirements
 
 The design requirements for this document are governed by three principal parameters:
 
 * to clearly mandate the design and implementation steps that are security critical and must be followed as is, while offering flexibility in the implementation of non-critical steps. In this manner, users can adapt to support different use models and deployment scenarios.
-* to delineate best practices to ensure that, should a vehicle be attacked, an attacker is forced to compromise many different systems.
-* to ensure that, if implemented, the security practices mandated or suggested in this document do not interfere with the functionality of ECUs, vehicles, or the manufacturing systems on which they run.
+* to delineate best practices to ensure that, should a vehicle be attacked, an attacker must compromise many different systems to access control or seriously impact functionality.
```

Comment:
"to access control" doesn't seem quite right to me. Would perhaps "to gain control" be better? Or "access data on the vehicle"? I think "access to" and "control of" a vehicle are two separate things, both of which we want to protect against. Maybe we should specify all the above: "to control the vehicle, access sensitive data, or seriously impact functionality".

Reply:
I liked your suggested expansion of the sentence and made the change.